RELEASE_NOTES.rst

```diff
@@ -842,7 +842,7 @@ https://github.com/coala-analyzer/coala/blob/v0.1.0-alpha/TUTORIAL.md
 
 If you want to write static code analysis routines, please check out
 this guide:
-https://github.com/coala-analyzer/coala/blob/v0.1.0-alpha/doc/getting\_involved/WRITING\_BEARS.md
+https://github.com/coala-analyzer/coala/blob/v0.1.0-alpha/doc/getting\_involved/WRITING\_Native\_BEARS.md
```

Comment:
Rename the file https://github.com/coala-analyzer/coala/blob/v0.1.0-alpha/doc/getting_involved/WRITING_BEARS.md first and I suggest you to change it to WRITING_NATIVE_BEARS instead of WRITING_Native_BEARS